spec/shortenator_spec.rb

```diff
@@ -30,13 +32,23 @@
   context '::search_and_shorten_links', :vcr do
     let(:original_text) { "text #{url}" }
     let(:url) { 'http://leafly.com' }
+    let(:original_args) { [original_text] }
+    let(:additonal_args) { [] }
 
-    subject { Shortenator.search_and_shorten_links(original_text) }
+    subject { Shortenator.search_and_shorten_links(*original_args, *additonal_args) }
```

Comment:
If you don't change `original_args` in any contexts, it's probably ok to pass `original_text` directly here, and skip a level of indirection.